src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -182,10 +198,50 @@ const ProjectInstallPlatform = createReactClass({
     );
   },
 
+  renderTestBody() {
+    let {integration, platform} = this.state;
+    let {dsnPublic} = this.props.platformData;
+
+    if (!integration || !platform) {
+      return <NotFound />;
+    }
+
+    analytics('experiment.installation_instructions', {
+      integration: integration.id,
+    });
+
+    return (
+      <Panel>
+        <PanelHeader hasButtons>
+          {t('Configure %(integration)s', {integration: integration.name})}
+          <Button size="small" href={integration.link} external>
+            {t('Full Documentation')}
+          </Button>
+        </PanelHeader>
+
+        <PanelBody disablePadding={false}>
+          {this.state.loading ? (
+            <LoadingIndicator />
+          ) : this.state.error ? (
+            <LoadingError onRetry={this.fetchData} />
+          ) : (
+            <InstallReactTest dsn={dsnPublic} />
+          )}
+        </PanelBody>
+      </Panel>
+    );
+  },
+
   render() {
+    let {installExperiment} = this.state;
+
     return (
       <div className="install row">
-        <div className="install-content col-md-10">{this.renderBody()}</div>
+        {installExperiment ? (
+          <div className="install-content col-md-10">{this.renderTestBody()}</div>
```

Comment:
small thing, but it looks like `<div className="install-content col-md-10">` is repeated, so maybe just put the ternary inside here?